data/kramdown-rfc2629.erb

```diff
@@ -12,14 +12,25 @@
 <% end -%>
 ]>
 
-<% ps.arr("pi", false) do |pi, val| -%>
-<?rfc <%=pi%>="<%= {true => "yes", false => "no", nil => "yes"}[val] || val %>"?>
+<%
+  rfcattrs = ps.attrs("ipr", "docName=docname", "category=cat",
+                      "consensus", "submissionType=submissiontype", "xml:lang=lang",
+                      "number", "obsoletes", "updates", "seriesNo=seriesno")
+  pis = KramdownRFC::ParameterSet.new({})
+  ps.arr("pi", false) do |pi, val|
+    pis[pi] = {nil => "true", false => "false", "true" => "true"}[val] || val
+  end
+  if piattrs = pis.attrs("tocDepth=tocdepth", "tocInclude=toc",
+                         "sortRefs=sortrefs", "symRefs=symrefs", "indexInclude=index")
+    rfcattrs += " " + piattrs
+  end
+  pis.rest.each do |pi, val|
+    v = {"true" => "yes", "false" => "no"}[val] || escape_html(val.to_s, :attribute)
```

Comment:
The cost of this mapping is that you can't set a PI to the string value "true" or "false".  I going to say that that is OK.

Reply:
I agree.